migrations/migrate-duplicate-emails.sql

```diff
@@ -0,0 +1,52 @@
+--------------------------------------------------------------------------------
+-- For each relation, find all records tied to a learn user that also have a
+-- github account with the same email, re-associate the records with the github
+-- account
+--------------------------------------------------------------------------------
+UPDATE membership
+  SET "user" = guser.id
+  FROM "user" luser
+  JOIN "user" guser
+    ON guser.email = luser.email
+  WHERE luser.id = membership.user
+    AND guser.plugin = 'github'
```

Comment:
This the condition clause, not the update. User records aren't changed as part of this process (at least, that's the intention). The relations (memberships, comments, and subscriptions) are re-assigned to the github user. That should leave all the (duplicate) learn records with no data associated with them, at which point they're deleted.